schemas/geodesyml/0.6/monumentInfo.xsd

```diff
@@ -179,7 +179,8 @@ Copyright: Scripps Orbit and Permanent Array Center (SOPAC), Commonwealth Govern
         <complexContent>
             <extension base="gml:AbstractGMLType">
                 <sequence>
-                    <element maxOccurs="unbounded" name="dataCenter" type="string"/>
+                    <element maxOccurs="unbounded" name="dataCenter" type="geo:DataCenterCodeType"/>
+	                <element minOccurs="0" maxOccurs="unbounded" name="gnssNetwork" type="geo:NetworkCodeType"/>
```

Comment:
This appears to be indented one too many times. Should be at same level as dataCenter.